src/06-loops.js

```diff
@@ -165,7 +222,16 @@ function timesIndex(array) {}
  *  //> [ 5, 5 + 2, 5 + 2 + 9 ]
  *  //> [ 5, 7, 16 ]
  */
-function cumulativeSum(array) {}
+ function cumulativeSum(array) {
+  let newArr = []
+  let won = array[0] 
+   for (let i = 0; i < array.length; i++){
```

Comment:
Lets take a closer look here.  How many times are we pushing the first index from `array` into `newArr`?